NEWS

```diff
@@ -1,6 +1,46 @@
 w3lib release notes
 ===================
 
+2.0.0 (to be released)
+----------------------
+
+Backwards incompatible changes:
+
+- Python 2 is no longer supported; Python 3.6+ is required now (#168, #175).
+- :func:`w3lib.url.safe_url_string` and :func:`w3lib.url.canonicalize_url`
+  no longer convert "%23" to "#" when it appears in URl path. This is a bug
+  fix. It's listed as a backwards incomatible change because in some
+  cases :func:`w3lib.url.canonicalize_url` output is going to change,
+  and so, if this output is used to generate URL fingerprints, fingerprints
+  might be incompatible with those created with the previous w3lib versions
+  (#141).
+
+Deprecation removals (#169):
+
+- ``w3lib.form`` module is removed.
+- ``w3lib.html.remove_entities`` function is removed.
+- ``w3lib.url.urljoin_rfc`` function is removed.
+
+The following functions are deprecated, and will be removed in future releases
+(#170):
+
+- `w3lib.util.str_to_unicode``
+- ``w3lib.util.unicode_to_str``
+- ``w3lib.util.to_native_str``
```

Comment:
These functions are not documented as `:func:...`, because they're not in w3lib.rst auto-generated API docs, and I didn't want to add them there.